skfuzzy/control/fuzzyvariable.py

```diff
@@ -13,8 +15,32 @@
 except ImportError:
     from .ordereddict import OrderedDict
 
+class TermPrimitive(object):
+    """Marker class used for type checking when a term
+    or term aggregate is expected"""
+    pass
 
-class FuzzyVariableTerm(object):
+    def membership_value(self):
```

Comment:
Should we still expose this here at all, if it's not implemented? Or is this an idea to be built on.